doc/releases/release-notes-3.7.rst

```diff
@@ -201,6 +201,10 @@ Networking
 
     * :c:func:`lwm2m_set_bulk`
 
+* IPSP:
+
+  * Removed IPSP support. ``CONFIG_NET_L2_BT`` does not exist anymore.
```

Comment:
This kconfig is not defined anymore. I didn't use the kconfig decorator so check_compliance doesn't yell at me.